server/handlers/custom-route-handler.js

```diff
@@ -13,6 +15,38 @@ exports.customRouteHandler = function customRouteHandler(req, res, next, { confi
       if(page.type === 'redirect') { 
         return res.redirect(page["status-code"], page["destination-path"]);
       }
+
+      if(page.type === 'static-page') {
+        if(page.metadata.header || page.metadata.footer) {
+          const modifiedThemeAttributes = Object.assign({}, config.config["theme-attributes"], { hide_breaking_news: true });
+          const updatedConfig = Object.assign({}, config.config, { "theme-attributes": modifiedThemeAttributes });
+          const store = createStore((state) => state, {
+            qt: {
+              pageType: page.type,
+              data: page,
+              config: updatedConfig,
+              currentPath: `${url.pathname}${url.search || ""}`,
+              disableIsomorphicComponent: true
+            }
+          });
+
+          const seoInstance = (typeof seo == 'function') ? seo(config) : seo;
+          const seoTags = seoInstance && seoInstance.getMetaTags(config, page.type, {}, {url});
+          
+          const cacheKeys = ["static"];
+          addCacheHeadersToResult(res, cacheKeys);
+
+          res.status(page["status-code"]);
```

Comment:
page["status-code"] || 200.
inline cacheKeys :-p
And you should probably add a cache header with the id of the static page. Probably x/publisher-id/page-id. Check with @AkashHC13 and @codepodu what to use, and also ask them to purge when the static page is updated